build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.3.2'
```

Comment:
I think this change isn't related to the "toggle" feature. It also introduces some lint issues and fixing those within this PR might make it more difficult to review. I believe it should be taken care of in its own PR. We always try to keep our PRs small and single-focused. Wdyt?

Reply:
True, I'll pull this out ASAP.

Reply:
👍